### ISSUE TITLE
bb should integrate with @apiaryio GitHub to provide information about what's being worked on

### ISSUE THREAD

**Issue by honzajavorek** [apiaryio/black-belt#188]
Heavily related to https://github.com/apiaryio/black-belt/issues/180#issuecomment-419386384 and https://github.com/apiaryio/black-belt/issues/189

-----

We decided we would document and track licensing issues [in a manner similar to Dredd](https://github.com/apiaryio/dredd/issues?q=is%3Aissue+is%3Aopen+label%3Alicensing), i.e. with a conventional titles and labeled, in all Apiary repos. Black belt could check the GitHub API for tracking issues and print out links to them in case the user is inspecting a dependency, so it would be clear to the user certain dependencies are known problems and somebody might be already working on them.

Basically, when printing the default output or the breadcrumbs and there are dependencies already tracked in Apiary issues, they should be linked for awareness.

**Comment by abtris** [comment]
Migrated to apiaryio/bb#11
